relay-filter/src/web_crawlers.rs

```diff
@@ -11,7 +11,7 @@ use crate::{FilterConfig, FilterStatKey};
 /// Checks if the event originates from a known web crawler.
 pub fn matches(event: &Event) -> bool {
     if let Some(user_agent) = user_agent::get_user_agent(event) {
-        WEB_CRAWLERS.is_match(user_agent)
+        !ALLOWED_WEB_CRAWLERS.is_match(user_agent) && WEB_CRAWLERS.is_match(user_agent)
```

Comment:
Let's swap this conditional such that for the majority of UAs, only one regex is executed.

Reply:
Done :)